src/compiler/reviews.md

```diff
@@ -1,4 +1,156 @@
-# Review policies
+Review Policy
+=============
+
+It is the purpose of code reviews to
+
+- **reduce** the risk of introducing **bugs** and usability and performance **regressions**,
+- keep our code **maintainable**, **readable**, and **documented**, and
+- ensure that changes are made with the big picture and **appropriate context in mind**.
+
+Reviewing accomplishes this by bringing in another set of eyes
+to look at a proposed change from a different perspective,
+which increases the chance of catching mistakes early and
+uncovering potential blind spots in the reasoning behind the change.
+
+
+## Basic Reviewing Requirements
+
+There are a number of requirements that need to be met in order for reviewing to be effective:
```

Comment:
one additional point that I have had trouble with (both as a reviewer and as someone looking for a reviewer) is that all of the points below were satisfied, but the reviewer was already swamped with reviews. What I'm trying to say is essentially "what do we do with PRs where no one can review it?".

Reply:
That indeed doesn't belong to this document.

I was rather thinking along the lines of "can we reject a PR due to insufficient reviewer capacity instead of letting it die slowly by not reviewing it"

Reply:
That's a good question -- and I have no good answer for it `:)` But it sounds like it would be a good bullet point for the guidance section.

Reply:
I've opened https://github.com/rust-lang/compiler-team/issues/448 for keeping track of this question.